fixtures/workers-with-assets-and-service-bindings/tests/index.test.ts

```diff
@@ -241,6 +245,20 @@ describe.each(devCmds)(
 						'env.NAMED_ENTRYPOINT.busyBee("🐝") response: Hello busy 🐝s from worker-d busyBee(bee)'
 					);
 				});
+
+				it("should support promise pipelining", async ({ expect }) => {
```

Comment:
Is this not going to fail?

Reply:
not for named entrypoints, since the Proxy Worker is added to the pipeline only for default exports